content/blog/2023-07-28-three-months-in-delft-two-open-hardware-projects/index.en.md

```diff
@@ -0,0 +1,184 @@
+
+The Academy however, is only a small part of the open hardware community in Delft.
+
+# Delft Open Hardware community
+
+The existing community of Delft Open Hardware is surrounded by people that are also working on optimizing their documentation for third part use. From this surrounding it is easy to obtain tips and ideas. Moreover, having a community that quickly provides feedback on the comprehensiveness and completeness of the documentation is crucial as it optimizes your work to many different professional backgrounds. Naturally, Delft’s highly diverse community, involving engineers, businesses, biologists, artists, and creative hobbyists fosters an environment for innovation. For instance, I met a developer whom decided to build an anti-theft system to prevent their bike from being stolen. This phone-size device is sensitive to vibrations, which can trigger the alarm to go off. This prompted me to determine if their hardware was also able to detect animal licks for my setup. The same technology is already implemented in some lick-detectors but have high financial cost. Additionally, the Delft community introduced me to a preliminary method for replacing the microstimulation setup’s most expensive component: the micromanipulator. The Open Hardware Automated microscope uses a lower-cost alternative to manipulate a platform, which could be adapted for microstimulation. By making hardware designs and documentation available to a wider audience, open hardware can foster a community of developers and enthusiasts who can work together to improve and enhance hardware systems.
+
+
+
+
+
+{{< card2 "OSH for the world">}} 
+
+Finally, I learned that open hardware documentation is not just important for hobbyists and enthusiasts. Open hardware documentation can also benefit commercial and industrial applications by enabling faster prototyping, reducing development costs, and facilitating innovation. By making hardware designs and documentation available to a wider audience, companies can tap into a diverse pool of talent and expertise and create more innovative and sustainable hardware products.
+
+{{< /card2 >}}
+
+#  Hardware work
+
+## Berlin time
+
+In November 2021, I started his master project that would use the microstimulation setup. However, the setup was no longer available in the lab. The people who worked on that paradigm had left the lab, too. Thus, I set out to re-establish the setup. My only source of information was an incomplete wiring diagram. Luckily, with the help of colleagues, I was able to get the basics of the setup to work: stimulate an electrode and detect licks. Next, I implemented the behavioural aquisition, meaning that I set up a camera that were able to clearly capture individual whisker movement and a camera that clearly captured the pupil of the animal. 
```

Comment:
In all honesty, Moritz was there to help a lot and he worked with the paradigm before maybe we should mention him explicitly.

Reply:
```suggestion
In November 2021, I started his master project that would use the microstimulation setup. However, the setup was no longer available in the lab. The people who worked on that paradigm had left the lab, too. Thus, I set out to re-establish the setup. My only source of information was an incomplete wiring diagram. Luckily, Moritz Drüke knew a lot about the setup and could help me. With his help, I was able to get the basics of the setup to work: stimulate an electrode and detect licks. Next, I implemented the behavioural aquisition, meaning that I set up a camera that were able to clearly capture individual whisker movement and a camera that clearly captured the pupil of the animal. 
```